### ISSUE TITLE
[Bug] Pull to Refresh isn't working properly on some websites

### ISSUE THREAD

**Issue by ghost** [mozilla-mobile/fenix#9724]
## Steps to reproduce
1. https://bing.com/covid
2. https://github.com/mozilla-mobile/fenix/issues
3. See the video I attached...?
Extract the ZIP file to see the video please. I had to compress it before uploading since Github wasn't accepting video files directly.

• #9720, #9718, #9715 seem to be talking about the issues related to pull to refresh as well. 

I was able to reproduce #9720, it is the same as the one appearing on: https://github.com/mozilla-mobile/fenix/issues as you can see in the video I attached, Pull to Refresh just doesn't work.

I wasn't able to reproduce #9718 Twitter works as expected on my device. 
 
Edit: Edited to make the wording more clearer.

### Expected behavior
Websites should react properly to Pull to Refresh just like Google Chrome does on mobile

### Actual behavior
Websites act weirdly when Pull to Refresh is used, they should work properly.

### Device information

* Android device: Motorola One Power, Android 10.
* Fenix version: Firefox Nightly 200404 18:01 ( Build #2015732627)

[Screen Record video.zip](https://github.com/mozilla-mobile/fenix/files/4433175/Screen.Record.video.zip)

**Comment by cadeyrn** [comment]
> #9720 #9718 #9715 seems to be talking about the same issue as this one

You forgot #9708. But why creating another issue if it's about the same problem? It's just another duplicate.

Anyway, #9715 and #9718 are definitively not the same as #9708 and #9720.

**Comment by SS1113** [comment]
> > #9720 #9718 #9715 seems to be talking about the same issue as this one
> 
> You forgot #9708. But why creating another issue if it's about the same problem? It's just another duplicate.
> 
> Anyway, #9715 and #9718 are definitively not the same as #9708 and #9720.

I'm sorry I tried to create another issue because I was facing the issue on another website. The reason why I mentioned other issues because those were talking about the issues comparable to mine but not exactly like the one I was facing the issue with; each one was talking about a specific website such as yours #9708 so I tried to unify it under one issue because it seems everyone seems to be having problems on different websites.

P.S. I didn't notice your issue earlier. That's why I didn't mention it.
